### PR TITLE
Added route-history controller

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | https://github.com/sulu/sulu-minimal/pull/43 https://github.com/sulu/sulu-standard/pull/814
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR adds the ability to view and delete history-routes.

#### Why?

This will be needed for the parent-page feature of the article-bundle.

### CODE REVIEW

File: src/Sulu/Bundle/RouteBundle/Controller/HistoryRouteController.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+use Sulu\Component\Rest\RestController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Provides api to handle routes.
+ *
+ * @RouteResource("history-routes")
+ */
+class HistoryRouteController extends RestController implements ClassResourceInterface
```
Comment (danrot) [inline src/Sulu/Bundle/RouteBundle/Controller/HistoryRouteController.php]: Shouldn't that be part of a `RouteController`? Having a `/history-routes` endpoint doesn't feel very restful to me... Something like `/nodes/uuid/routes?history=true` would suit better IMO, however every bundle would have to include that Controller as a subresource then... Not sure about that yet, but I think that would make sense.

Comment (wachterjohannes) [inline src/Sulu/Bundle/RouteBundle/Controller/HistoryRouteController.php]: i think i will refactor it to be `/admin/api/routes?history=true`. the other would work when we make the controller as a service. but in the template, the URL would differ for each type of content.

File: src/Sulu/Bundle/RouteBundle/Resources/views/Template/content-types/route.html.twig [hunk 2]
```diff
@@ -5,6 +5,9 @@
      data-form="true"
      data-type="resourceLocator"
      data-aura-component="resource-locator@sulucontent"
+     data-aura-history-api="/admin/api/history-routes?entityClass=<%= entityClass %>&entityId=<%= entityId %>&locale={{ languageCode }}"
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/RouteBundle/Resources/views/Template/content-types/route.html.twig]: see here

**Comment by wachterjohannes** [comment]
@danrot fixed

**Pull request merged**